cmd/ndm_daemonset/probe/blkidprobe.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2020 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package probe
```

Comment:
@cospotato Do we need a new probe for fetching this details? Can this be included in some existing probes? WDYT?

Reply:
@akhilerm I think it's need this new probe to get partition table UUID by blkid. Because only the `udevprobe` can provide partition table UUID in the existing probes. But on `CentOS 7`, the host udev did not provide it. But we use the cache from host udev. So we cannot get partition table UUID with udev on `CentOS 7` host.